apis/Google.Cloud.Speech.V1Beta1/Google.Cloud.Speech.V1Beta1/SpeechClient.cs

```diff
@@ -570,16 +582,27 @@ public virtual OperationsClient LongRunningOperationsClient
                 LongRunningOperationsClient,
                 callSettings);
 
+        public abstract class StreamingRecognizeStream : BidirectionalStreamingBase<StreamingRecognizeRequest, StreamingRecognizeResponse>
+        {
+        }
+
+        public virtual StreamingRecognizeStream StreamingRecognize(
```

Comment:
Any reason there isn't an async version of this? It's making a network call, isn't it? (Will make an async equivalent in my local copy.)